### PR TITLE
Use ZWSP instead of WBR

### PR DESCRIPTION
**Pull request by tushuhei**
This changes the default separator from WBR to ZWSP.

We observed that some screen readers split their text focus by chunks because of the inserted WBR elements. We also considered adding the `aria-hidden="true"` attribute to the WBR elements as a workaround, but the behavior was still not ideal and stable. On the other hand, the ZWSP approach does not split the text focus and works better because it does not alter the DOM tree structure. Considering that, I suggest to use ZWSP as the default separator for accessibility improvement.

Users can still use WBR tags if it's preferred by specifying the separator element.

```typescript
import {jaModel, HTMLProcessingParser} from 'budoux';
const parser = HTMLProcessingParser(jaModel, {separator: document.createElement('wbr')});
```

### CODE REVIEW

File: java/src/main/java/com/google/budoux/HTMLProcessor.java [hunk 1]
```diff
@@ -126,9 +139,9 @@ public void tail(Node node, int depth) {
    * @param html the HTML string to resolve.
    * @return the HTML string of phrases wrapped in non-breaking markup.
    */
-  public static String resolve(List<String> phrases, String html) {
+  public static String resolve(List<String> phrases, String html, String separator) {
```
Comment (kojiishi) [inline java/src/main/java/com/google/budoux/HTMLProcessor.java:142]: nit: This is a public API, is it ok to change without making `separator` an optional argument?

**Review by kojiishi — APPROVED** [review]
LGTM w/nit, thanks!

File: java/src/main/java/com/google/budoux/HTMLProcessor.java (hunk 1 above)
Comment (tushuhei) [reply java/src/main/java/com/google/budoux/HTMLProcessor.java:154]: That's a good point! I added the old API for backward compatibility.

**Comment by tushuhei** [comment]
Thanks for reviewing!

**Pull request merged**